companion/src/modeledit/setup.cpp

```diff
@@ -238,9 +238,9 @@ ModulePanel::ModulePanel(QWidget * parent, ModelData & model, ModuleData & modul
   lock = false;
 
   // a change will not register if only one item was added
-  if (ui->protocol->count() == 1)
+  if (ui->protocol->count() == 1) {
```

Comment:
Does this mean we should always use brackets even for single-line ifs?  Our previous discussion on the matter seemed to suggest otherwise (in certain circumstances).  I'm fine with it either way, just looking for consistency.